packages/crossweb-pl/src/Meetup/CrosswebPlMeetupFactory.php

```diff
@@ -36,10 +37,12 @@ public function createMeetupFromMeetupUrl(string $url, string $name): ?Meetup
 
         $groupName = $this->resolveGroupName($name);
 
+        $daysUntil = Carbon::parse(date('Y-m-d'))->diffInDays($startDateTime);
         return new Meetup(
             $name,
             $groupName,
             $startDateTime,
+            $daysUntil,
```

Comment:
This error:

```bash
Missing data of "$daysUntil" parameter for hydrated class "Fop\Meetup\Value  
!!    Object\Meetup" __construct method.
```

means that data is missing in generated file:
https://github.com/TomasVotruba/friendsofphp.org/blob/main/config/_data/meetups.php#L19-L28

The `$daysUntil` must be defined only inside object, not in its constructor.